### ISSUE TITLE
Jaws does not read links in an aria-describedby

### ISSUE THREAD

**Issue by 7dITS** [FreedomScientific/VFO-standards-support#213]
### Summary

> Links within an aria-describedby are not read when navigated to using shortcut keys. The link will read as a link when tabbing and arrowing, but will not read as a link when it is called as describedby text (the associated control is focused). 

I need to know if this is a defect or if the code I provided needs to be redone. 
Thank you in advance.
> 
> Example:
> 1. Navigate to edit field using the E shortcut key,
> 2. Listen to the edit field that contains a description with a link in it.

### Expected result

> What did you expect to happen:
I should hear link [link name] opens new window.

### Actual result

> What actually did happen?
When I tab or arrow to the links located in the aria-describedby links are announced. When I try to use the short cut key of E or U they are not announced as links just regular text. 
### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
![image](https://user-images.githubusercontent.com/24323027/60831928-b6711800-a188-11e9-9dc3-753ae7ee60e8.png)

#### JAWS version and build number
JAWS 2019 2019.1906.10.400 and NVDA 2019.1.1

 
#### Operating System and version
Windows 10 Enterprise 1803

#### Browser and version:
Internet explorer 11 Version: 11.0.130
and Fire fox ESR 60.7.2esr (64-bit)

**Comment by JAWS-test** [comment]
This is not a defect, but correct according to the W3C specifications. A label (per label or aria-labelledby) or description (per aria-describedby) is output as plain text. The role of interactive elements within the label or description should not be output. See: https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te

**Comment by 7dITS** [comment]
@JAWS-test  Thank you for the information.
